plugins/statistics.koplugin/main.lua

```diff
@@ -205,7 +202,11 @@ function ReaderStatistics:initData()
 end
 
 function ReaderStatistics:isEnabled()
-    return not self:isDocless() and self.settings.is_enabled
+    return self.settings.is_enabled and self.is_doc
+end
+
+function ReaderStatistics:isEnabledNotFreezed()
```

Comment:
Nit: `Frozen` ;). (ditto for the variable)

(I'd possibly add an `And` in the middle there, too).